x-pack/plugins/maps/public/components/map/mb/view.js

```diff
@@ -9,20 +9,29 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { ResizeChecker } from 'ui/resize_checker';
 import { syncLayerOrder, removeOrphanedSourcesAndLayers, createMbMapInstance } from './utils';
-import { DECIMAL_DEGREES_PRECISION, ZOOM_PRECISION } from '../../../../common/constants';
+import { DECIMAL_DEGREES_PRECISION, FEATURE_ID_PROPERTY_NAME, ZOOM_PRECISION } from '../../../../common/constants';
 import mapboxgl from 'mapbox-gl';
 import { FeatureTooltip } from '../feature_tooltip';
 
+
+const TOOLTIP_TYPE = {
```

Comment:
These are constants used by the store and actions. I think they should be created in a more global location under `public/store`

Reply:
theoretically yes, in practice no. the tooltip-state is only evaluated here to handle moves/clicks correctly. I would move these constants once they are actually being evaluated in the redux flow (e.g, tooltip type will be relevant when trying to preserve tooltips across data-refreshes, but it's not something addressed here).